lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -853,26 +853,8 @@ class SemanticsObject {
         hasIdentityTransform &&
         verticalContainerAdjustment == 0.0 &&
         horizontalContainerAdjustment == 0.0) {
-      if (isDesktop) {
-        element.style
-          ..removeProperty('transform-origin')
-          ..removeProperty('transform');
-      } else {
-        element.style
-          ..removeProperty('top')
-          ..removeProperty('left');
-      }
-      if (containerElement != null) {
-        if (isDesktop) {
-          containerElement.style
-            ..removeProperty('transform-origin')
-            ..removeProperty('transform');
-        } else {
-          containerElement.style
-            ..removeProperty('top')
-            ..removeProperty('left');
-        }
-      }
+      _resetElementOffsets();
+      _resetContainerOffsets(containerElement);
```

Comment:
It looks like these two methods do the exact same thing but on different html elements. Maybe consolidate them into one method that takes the element as an argument?

Reply:
Thanks. Done.